### PR TITLE
Fix key missmatch for consoles: user -> username

### PR DESCRIPTION
**Pull request by asdil12**
See https://progress.opensuse.org/issues/104571

### CODE REVIEW

**Comment by asdil12** [comment]
openqa-clone-custom-git-refspec https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14008 https://openqa.opensuse.org/tests/2133066
Created job #2135444: opensuse-Tumbleweed-JeOS-for-RPi-aarch64-Build20220111-jeos-containers@RPi3 -> https://openqa.opensuse.org/t2135444

File: tests/containers/rootless_podman.pm [hunk 1]
```diff
@@ -56,6 +56,7 @@ sub run {
     # already exists owned by root
     assert_script_run 'rm -rf /tmp/script*';
     ensure_serialdev_permissions;
+    assert_script_run 'loginctl enable-linger 1000';
```
Comment (dcermak) [inline tests/containers/rootless_podman.pm:59]: Better safe than sorry :wink: 
```suggestion
    assert_script_run 'loginctl enable-linger $(id -u bernhard)';
```

**Comment by Vogtinator** [comment]
IMO openQA should ssh as bernhard, which should start a session just like a local login.

File: tests/containers/rootless_podman.pm (hunk 1 above)
Comment (b10n1k) [reply tests/containers/rootless_podman.pm:59]: even better if _bernhard_ is replaced by `$user`

**Comment by asdil12** [comment]
Let's see what will happen

openqa-clone-custom-git-refspec https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14008 https://openqa.opensuse.org/tests/2133066
Created job #2136345: opensuse-Tumbleweed-JeOS-for-RPi-aarch64-Build20220111-jeos-containers@RPi3 -> https://openqa.opensuse.org/t2136345

**Comment by Vogtinator** [comment]
> Let's see what will happen

If that works, please also fix the other typos.

BTW: I see some `Use of uninitialized value $os_release{"VERSION"}` in the log

**Comment by Vogtinator** [comment]
LGTM, just the commit/PR subject/message should mention that this is basically just fixing a key name mismatch

**Comment by jlausuch** [comment]
> LGTM, just the commit/PR subject/message should mention that this is basically just fixing a key name mismatch

yeah, exactly. Maybe even no need to mention rootles_podman in the commit message.

**Comment by asdil12** [comment]
done

**Review by Vogtinator — APPROVED** [review]

**Review by jlausuch — APPROVED** [review]

**Review by mloviska — APPROVED** [review]

**Pull request merged**